testsuite/MDAnalysisTests/core/test_universe.py

```diff
@@ -210,6 +210,10 @@ def test_universe_topology_class_with_coords(self):
         assert_equal(u.trajectory.n_frames, u2.trajectory.n_frames)
         assert u2._topology is u._topology
 
+    def test_universe_empty_ROMol(self):
+        Chem = pytest.importorskip("rdkit.Chem")
+        mol = Chem.Mol()
+        u = mda.Universe(mol, format="RDKIT")
```

Comment:
I suppose you could also check that the universe is empty--has `0` atoms for example.

Reply:
Yes, I included the asserts also.